beescale-yun-2022.ino

```diff
@@ -1,12 +1,25 @@
-/*
-  Yun Open Bee Hive Scale GET-Sketch
-  2015-01 by mois, more: http://wbk.in-berlin.de/wp/blog/series/bienenwaage/
-  2015-03 modified by Alexander Wilms
-  Dieser Sketch von mois ist lizensiert unter einer Creative Commons Namensnennung - Nicht-kommerziell - Weitergabe unter gleichen Bedingungen 4.0 International Lizenz.
-  This code by mois is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License.
+/**
+
+  Mois Labs Beescale Yun - Firmware source code
+  Collect sensor data, send it via GSM to local SD, flat-file on webserver, and to Hiveeyes' Kotori/InfluxDB/Grafana.
+
+  https://hiveeyes.org/docs/arduino/firmware/moislabs/beescale-yun/README.html
+
+  (c) 2011-2023 Markus Euskirchen
+  License: Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License
+  Lizenz:  Creative Commons Namensnennung - Nicht-kommerziell - Weitergabe unter gleichen Bedingungen 4.0 International Lizenz
+
+  Source code repository and documentation:
+  - https://github.com/bee-mois/beescale
+  - https://github.com/hiveeyes/arduino/tree/main/moislabs/beescale-yun
+  - https://www.euse.de/wp/blog/series/bienenwaage2/
 
   Changelog:
-  2015-03-14 Alexander Wilms - Modularized code into functions, adjusted loop order to avoid program hangs in case of client/network/transmit errors. Code cleanup.
+  2014-04-04 mois  - Initial version
+  2015-01-12 mois  - Added second scale
+  2015-03-14 Alexander Wilms -
+                     Modularized code into functions, adjusted loop order to avoid program hangs in case of
+                     client/network/transmit errors. Code cleanup.
```

Comment:
Those are the manual changes I added on top of the automatic code formatting. Let me know if you want to see any adjustments here.